acceptance/static_creds_test.go

```diff
@@ -0,0 +1,175 @@
+//go:build acceptance
+// +build acceptance
+
+package acceptance
+
+import (
+	"fmt"
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/openstack/identity/v3/roles"
+	"github.com/gophercloud/gophercloud/openstack/identity/v3/users"
+	"github.com/opentelekomcloud/vault-plugin-secrets-openstack/openstack"
+	"github.com/opentelekomcloud/vault-plugin-secrets-openstack/openstack/fixtures"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"net/http"
+	"testing"
+)
+
+type testStaticCase struct {
+	Cloud      string
+	ProjectID  string
+	DomainID   string
+	Root       bool
+	SecretType string
+	Username   string
+	Extensions map[string]interface{}
+}
+
+func (p *PluginTest) TestStaticCredsLifecycle() {
+	t := p.T()
+
+	cloud := openstackCloudConfig(t)
+	require.NotEmpty(t, cloud)
+
+	client, aux := openstackClient(t)
+
+	userRoleName := "member"
+
+	dataCloud := map[string]interface{}{
+		"auth_url":         cloud.AuthURL,
+		"username":         cloud.Username,
+		"password":         cloud.Password,
+		"user_domain_name": cloud.UserDomainName,
+	}
+
+	cases := map[string]testStaticCase{
+		"user_password": {
+			Cloud:      cloud.Name,
+			ProjectID:  aux.ProjectID,
+			DomainID:   aux.DomainID,
+			Username:   "static-test-1",
+			SecretType: "password",
+		},
+		"user_token": {
+			Cloud:      cloud.Name,
+			ProjectID:  aux.ProjectID,
+			DomainID:   aux.DomainID,
+			Username:   "static-test-2",
+			SecretType: "token",
+			Extensions: map[string]interface{}{
+				"identity_api_version": "3",
+			},
+		},
+	}
+
+	for name, data := range cases {
+		t.Run(name, func(t *testing.T) {
+			data := data
```

Comment:
Optional: this re-assignment makes sense with `t.Parallel` mostly, you can add it

Reply:
This test accesses the real cloud user and t.Parallel will lead to conflict and error.